### PR TITLE
Add Chromebook specific properties to Skylake and newer config.plist's

### PR DESCRIPTION
**Pull request by meghan06**
Added `ProtectMemoryRegions` under ACPI -> Booter -> Quirks to fix shutdown and restart on coreboot UEFI firmware that Chromebooks utilize and  `-igfxnotelemetryload` to fix iGPU acceleration on 10.15 and up.

### CODE REVIEW

**Comment by meghan06** [comment]
I'm PR #420 nice

File: config-laptop.plist/coffee-lake-plus.md [hunk 1]
```diff
@@ -553,6 +557,7 @@ System Integrity Protection bitmask
 | boot-args | Description |
 | :--- | :--- |
 | **-wegnoegpu** | Used for disabling all other GPUs than the integrated Intel iGPU, useful for those wanting to run newer versions of macOS where their dGPU isn't supported |
+|**-igfxnotelemetryload** | Prevents iGPU telemetry from loading. iGPU telemetry may cause a freeze during startup on certain laptops such as Chromebooks on macOS 10.15 and higher, see [here](https://github.com/acidanthera/WhateverGreen#intel-hd-graphics) for more information.
```
Comment (1Revenger1) [inline config-laptop.plist/coffee-lake-plus.md:560]:
```suggestion
| **-igfxnotelemetryload** | Prevents iGPU telemetry from loading. iGPU telemetry may cause a freeze during startup on certain laptops such as Chromebooks on macOS 10.15 and higher, see [here](https://github.com/acidanthera/WhateverGreen#intel-hd-graphics) for more information.
```
Needs to be fixed in the other files too.

**Comment by 1Revenger1** [comment]
Double check the Lint output, looks like there is a couple small fixes you need to apply too. It generated a diff you can apply.

**Comment by meghan06** [comment]
@1Revenger1 Looks alright to me!

File: config-laptop.plist/coffee-lake.md [hunk 2]
```diff
@@ -522,6 +526,7 @@ System Integrity Protection bitmask
 | boot-args | Description |
 | :--- | :--- |
 | **-wegnoegpu** | Used for disabling all other GPUs than the integrated Intel iGPU, useful for those wanting to run newer versions of macOS where their dGPU isn't supported |
+|**-igfxnotelemetryload** | Prevents iGPU telemetry from loading. iGPU telemetry may cause a freeze during startup on certain laptops such as Chromebooks on macOS 10.15 and higher, see [here](https://github.com/acidanthera/WhateverGreen#intel-hd-graphics) for more information.
```
Comment (1Revenger1) [inline config-laptop.plist/coffee-lake.md:529]:
```suggestion
| **-igfxnotelemetryload** | Prevents iGPU telemetry from loading. iGPU telemetry may cause a freeze during startup on certain laptops such as Chromebooks on macOS 10.15 and higher, see [here](https://github.com/acidanthera/WhateverGreen#intel-hd-graphics) for more information.
```

Comment (1Revenger1) [reply config-laptop.plist/coffee-lake.md:529]: It's good to go after this fix.

**Comment by meghan06** [comment]
@1Revenger1 Complete.

**Review by 1Revenger1 — APPROVED** [review]

**Pull request merged**